Source/carthage/CopyFrameworks.swift

```diff
@@ -24,11 +24,11 @@ public struct CopyFrameworksCommand: CommandType {
 				|> flatMap(.Concat) { frameworkPath -> SignalProducer<(), CarthageError> in
 					let frameworkName = frameworkPath.lastPathComponent
 
-					let source = NSURL(fileURLWithPath: frameworkPath, isDirectory: true)!
+					let source = Result(NSURL(fileURLWithPath: frameworkPath, isDirectory: true), failWith: CarthageError.InvalidArgument(description: "Could not find framework \"\(frameworkName)\" at path \(frameworkPath). Ensure that the given path is approriately entered. Ensure that your \"Input Files\" have been entered correctly."))
```

Comment:
I would combine the "Ensure that" sentences. ("…entered and that your…").

Also, there's a typo in _appropriately_.